src/current/_includes/releases/v24.3/feature-highlights-cloud.html

```diff
@@ -0,0 +1,47 @@
+<table>
```

Comment:
```suggestion
<table markdown="1">
```

@shannonbradshaw So that the markdown link `[CockroachDB Cloud](https://cockroachlabs.cloud/signup)` in this table will render properly.